contracts/BitGuildFeeOracle.sol

```diff
@@ -0,0 +1,53 @@
+pragma solidity ^0.4.24;
+
+import "./BitGuildAccessAdmin.sol";
+
+
+/**
+ * @title BitGuildFeeOracle
+ * @dev Fee definition
+ */
+contract BitGuildFeeOracle is BitGuildAccessAdmin {
```

Comment:
I wouldn't call it an oracle btw. Oracles as a term is for when there's external data feeding into the smart contracts. Let's just call it BitGuildMarketplaceFeeProvider